examples/lfric/eg14/Makefile

```diff
@@ -31,34 +31,86 @@
 # ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
 # POSSIBILITY OF SUCH DAMAGE.
 # ------------------------------------------------------------------------------
-# Author: A. R. Porter, STFC Daresbury Laboratory
-# Modified J. Henrichs, Bureau of Meteorology
+# Author: J. Henrichs, Bureau of Meteorology
+# Modifications: A. R. Porter, STFC Daresbury Laboratory
 
-include ../../common.mk
+# The compiler to use may be specified via the F90 environment
+#
+# export F90=gfortran
+# export F90FLAGS="-Wall -g -fcheck=bound"
+
+PSYROOT=../../..
+
+include $(PSYROOT)/examples/common.mk
+
+GENERATED_FILES = *.o *.mod  $(EXEC) main_alg.f90 main_psy.f90
+
+F90 ?= gfortran
+F90FLAGS ?= -Wall -g
+
+OBJ = main_psy.o main_alg.o testkern_w0_kernel_mod.o
+
+EXEC = example_openacc
+
+LFRIC_DIR ?= $(PSYROOT)/src/psyclone/tests/test_files/dynamo0p3/infrastructure
+LFRIC_NAME=lfric
+LFRIC_LIB=$(LFRIC_DIR)/lib$(LFRIC_NAME).a
+
+F90FLAGS += -I$(LFRIC_DIR)
+
+.PHONY: transformtransform_kernels transform_parallel transform_parallel_dm
+
+transform: transform_kernels transform_parallel transform_parallel_dm
 
-.PHONY: kernels parallel_no_dm parallel_dm
+transform_kernels:
+	${PSYCLONE} -nodm -s ./acc_kernels.py \
+	-opsy main_psy.f90 -oalg main_alg.f90 main.x90
 
-transform: kernels parallel_no_dm parallel_dm
+transform_parallel:
+	${PSYCLONE} -nodm -s ./acc_parallel.py \
+	-opsy main_psy.f90 -oalg main_alg.f90 main.x90
 
-kernels:
-	${PSYCLONE} -s ./acc_kernels.py -nodm \
-  ../code/gw_mixed_schur_preconditioner_alg_mod.x90 \
-  -oalg /dev/null
+transform_parallel_dm:
```

Comment:
Please put a TODO for #1669 here as the generated code is currently no correct.

Reply:
Done. You are aware that there are also big problems with builtins?
```
!$acc& copyin(nlayers,field1_proxy,field1_proxy%data,0.0_r_def,ndf_aspc1_field1,undf_aspc1_field1,map_aspc1_field1,field2_proxy, &
!$acc& field2_proxy%data,1.0_r_def,ndf_aspc1_field2,undf_aspc1_field2,map_aspc1_field2)
      !
      !$acc parallel default(present)
      !$acc loop independent
      DO df=loop0_start,loop0_stop
        field1_proxy%data(df) = 0.0_r_def
      END DO
```
nlayers, (u)ndf... etc are all undefined in case of a builtin.

Reply:
Oh dear, no, I wasn't aware of that. If you could open an Issue that would be great.

Reply:
#1694 
I've added a todo to the Makefile, as an explanation for why compilation fails. Couldn't think of a better place :)